src/mk.sh

```diff
@@ -489,6 +503,26 @@ function kernel_deploy
   fi
 }
 
+# This function retrieves and prints information related to the building
+# process of the kernel.
```

Comment:
How about:

> This function retrieves and prints information related to the kernel that will be compiled.

In some way, "building process" sounds weird because the sort of information we show is not related to the building process.